CMakeLists.txt

```diff
@@ -154,6 +163,59 @@ if(NOT CMAKE_INSTALL_RPATH AND BUILD_SHARED_LIBS)
   set(CMAKE_INSTALL_RPATH ${base} ${base}/${relDir})
 endif()
 
+set(STD_FS_LIB "")
+# ==== The test below is based on:
```

Comment:
* Yes, RHEL 8 (having gcc 8.x ) will fail without this.
* Not tested personaly with oother combinations outside fedora/rhel products.

A well proven routine is used from pybind11 library.

Reply:
OK, I can merge this for now, but please be aware that I don't have the ability to maintain the codebase for very old compilers and so the build may or may not work for GCC 8 (and as mentioned I will be raising the minimum required C++ version to 20 early in 2023) and may be broken by any future changes that I make. Even the GCC team does not support GCC 8 anymore.

Reply:
* Currently rhel8 is fine, in case you start require c++20, that's it, will drop out rhel8 support.
* There would be still some releng way to invoke gcc 10 even on RHEL 8, will update packaging at that time.

If interested in rhel/fedora native repo having sv-lang the automated (weekly) [builds are here](https://copr.fedorainfracloud.org/coprs/rezso/HDL/builds/) .